### DISCUSSION TITLE
命令行无法导入

### DISCUSSION THREAD

**Discussion by inkss** [ArtalkJS/Artalk#105] [Q&A] [answered]
```
FATAL `数据行囊` 文件不存在，请检查路径是否正确
```

```
./artalk-go --config config.yml import twikoo t_name:"枋柚梓的猫会发光" t_url:"https://inkss.cn" json_file:"database.json"
```

**Comment by inkss** [comment]
还有前端导入也有点问题，匹配的不对呀

![原生 JS 的自定义右键实现 - 枋柚梓的猫会发光 和另外 3 个页面 - 个人 - Microso_22-04-17_11-22-57](https://user-images.githubusercontent.com/31947043/163698906-eef00170-b916-47a4-86dc-df13fc21cb46.png)


![Inkss – sites  typecho@SH _22-04-17_11-22-08](https://user-images.githubusercontent.com/31947043/163698897-eab10f17-4747-476d-b000-dbe4cfe6cf99.png)

**Reply by qwqcode** [reply]
> 还有前端导入也有点问题，匹配的不对呀
> 
> ![原生 JS 的自定义右键实现 - 枋柚梓的猫会发光 和另外 3 个页面 - 个人 - Microso_22-04-17_11-22-57](https://user-images.githubusercontent.com/31947043/163698906-eef00170-b916-47a4-86dc-df13fc21cb46.png)
> 
> ![Inkss – sites typecho@SH _22-04-17_11-22-08](https://user-images.githubusercontent.com/31947043/163698897-eab10f17-4747-476d-b000-dbe4cfe6cf99.png)

没看出问题

**Reply by inkss** [reply]
导入的不对呀，我输入的了目标站点/地址，然后看结果怎么在sites表新建了一条记录，全部都没匹配上

**Comment by qwqcode** [comment]
https://artalk.js.org/guide/transfer.html

**Reply by inkss** [reply]
so？

**Comment by qwqcode** [comment, marked as answer]
😅 确实存在问题，正在修

**Reply by inkss** [reply]
哈哈哈哈哈哈哈哈哈，好的

**Reply by qwqcode** [reply]
好了，后端升级到 v2.1.1/0c69206

**Reply by inkss** [reply]
前端要不要升级？对了那个pv是怎么用的？手动指定 pvEl 吗？

**Reply by inkss** [reply]
这里的链接地址得更新了

https://artalk.js.org/guide/frontend/config.html#%E5%9F%BA%E6%9C%AC%E9%85%8D%E7%BD%AE-%E5%BF%85%E5%A1%AB%E9%A1%B9

![🔭 配置 · 前端  Artalk 和另外 6 个页面 - 个人 - Microsoft​ Edg_22-04-17_18-43-32](https://user-images.githubusercontent.com/31947043/163711056-7fc10588-1a7f-494d-b9e3-bcdd7ac28d4f.png)

**Comment by inkss** [comment]
康康

![Termius - SH-2-4-6_22-04-17_18-55-08](https://user-images.githubusercontent.com/31947043/163711455-6db41271-2b2a-4380-bfdf-50c5bacb5b83.png)

**Reply by qwqcode** [reply]
转换工具转换后导入artrans格式的文件，artalk-go import 不加 twikoo

**Comment by inkss** [comment]
不过现在用前端导入正常了，就是还是有499.
![Ai - SH 和另外 3 个页面 - 个人 - Microsoft​ Edge_22-04-17_19-08-24](https://user-images.githubusercontent.com/31947043/163711830-d5770ac5-341a-47c0-83e7-1697c5bdfbfe.png)

**Comment by inkss** [comment]
咳咳，还是得来麻烦你，数据是导进去了，但是不显示呀，复现地址：https://inkss.cn/page/info/ 

而且，我看数据库怎么存的是：
![Inkss – pages  Tencent@SH _22-04-17_19-29-58](https://user-images.githubusercontent.com/31947043/163712449-796042ea-be92-4367-bc29-a499a1f68b55.png)

![Inkss – comments  Tencent@SH _22-04-17_19-31-02](https://user-images.githubusercontent.com/31947043/163712479-ce91280e-a224-4356-b9ea-c93bf7306d79.png)

原始数据里不是呀：

![database artrans - Hexo-Blog - Visual Studio Code_22-04-17_19-31-38](https://user-images.githubusercontent.com/31947043/163712501-c92fe222-8f3b-49b1-854f-36cf0498fe95.png)

而且，即使我访问 https://inkss.cn/page/info 也是没有数据

**Reply by qwqcode** [reply]
> 咳咳，还是得来麻烦你，数据是导进去了，但是不显示呀，复现地址：https://inkss.cn/page/info/
> 
> 而且，我看数据库怎么存的是： ![Inkss – pages Tencent@SH _22-04-17_19-29-58](https://user-images.githubusercontent.com/31947043/163712449-796042ea-be92-4367-bc29-a499a1f68b55.png)
> 
> ![Inkss – comments Tencent@SH _22-04-17_19-31-02](https://user-images.githubusercontent.com/31947043/163712479-ce91280e-a224-4356-b9ea-c93bf7306d79.png)
> 
> 原始数据里不是呀：
> 
> ![database artrans - Hexo-Blog - Visual Studio Code_22-04-17_19-31-38](https://user-images.githubusercontent.com/31947043/163712501-c92fe222-8f3b-49b1-854f-36cf0498fe95.png)
> 
> 而且，即使我访问 https://inkss.cn/page/info 也是没有数据

这个问题还存在吗？

**Reply by inkss** [reply]
我现在是用前端导入的，导进去了
